ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/EditorWithAutoSave.java

```diff
@@ -11,9 +11,24 @@
 package org.eclipse.che.ide.api.editor;
 
 /**
+ * Editor auto save functionality.
+ * It's supports enable/disable auto save.
  * @author Evgen Vidolob
  */
 public interface EditorWithAutoSave {
 
+    /**
+     * @return true if auto save is enabled, false otherwise.
```

Comment:
See [The summary fragment -> Tip](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment). Actually i am working on che code style spec and it is the same in this case.